### DISCUSSION TITLE
Add `--python-path` command line option

### DISCUSSION THREAD

**Discussion by paul-marechal** [UPBGE/upbge#938] [Ideas/Features for UPBGE 0.2.5+]
I recently discovered that there is a way to run the bundled Python interpreter if you only have access to the upbge executable, by doing `upbge -b --python-console`. We can also execute scripts and expressions directly.

But for some tools, it would be required to get the path to the Python interpreter used, and while there might be ways using the previously mentioned options, it is not very user friendly.

**Request:**
`upbge -b --python-path` would only output the path to the Python interpreter used by upbge.

Either have this option on the main upbge executable or on the game player.

**Comment by youle31** [comment]
I move this to discussions
